tests/unit/NameRequestInfo.spec.ts

```diff
@@ -215,7 +215,39 @@ describe('Name Request Info with a NR', () => {
     expect(listItems[5].textContent).toContain('Condition/Consent: Not Received')
   })
 })
+describe('Numbered Amalgamation Info component', () => {
+  let wrapper: any
+
+  beforeEach(() => {
+    store.stateModel.tombstone.filingType = FilingTypes.AMALGAMATION
+    store.stateModel.tempId = 'T1234567'
+    store.stateModel.entityType = CorpTypeCd.BENEFIT_COMPANY
+    store.stateModel.nameRequest.nrNum = null
+    store.stateModel.nameRequestApprovedName = null
+    wrapper = mount(NameRequestInfo, { vuetify })
+  })
 
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('renders numbered company info', () => {
+    expect(wrapper.vm.$el.querySelector('#numbered-amalgamation-info').textContent)
+      .toContain('Resulting Business Name')
+
+    expect(wrapper.vm.$el.querySelector('.numbered-amalgamation-list-items')).toBeDefined()
+  })
+
+  it('renders the numbered company content', () => {
+    const listItems = wrapper.vm.$el.querySelectorAll('.numbered-amalgamation-list-items li')
+    expect(listItems.length).toEqual(4)
+
+    expect(listItems[0].textContent).toContain('[Incorporation Number] B.C. LTD.')
+    expect(listItems[1].textContent).toContain('You will be filing this Incorporation Application')
+    expect(listItems[2].textContent).toContain('Your Incorporation Number will be generated at the end')
+    expect(listItems[3].textContent).toContain('It is not possible to request a specific Incorporation Number')
+  })
+})
 describe('Name Request Info component without a NR', () => {
```

Comment:
^^ beautiful!

Also please add a blank line between the previous block and this new describe block.